Vagrantfile

```diff
@@ -20,10 +20,16 @@ Vagrant.configure(2) do |config|
 
   config.ssh.forward_x11 = true
 
-  config.vm.provision 'shell',
-                      path: 'bin/bootstrap',
-                      privileged: false,
-                      keep_color: true
+  config.vm.provision "ansible" do |ansible|
+    ansible.playbook = "deploy/opencraft.yml"
+    ansible.raw_arguments = ["--extra-vars", "@deploy/vagrant_vars.yml"]
+    if FileTest.exists?("deploy/private.yml")
```

Comment:
Sorry I'm not so good with Ruby but could we loop through these as a list of `['deploy/private.yml', 'private.yml']`?

Reply:
I'm not overly familiar with Ruby either, so tried to keep it simple, but I think looping over it makes sense, will actually be simpler.

Reply:
Your solution looks fine to me, as long it runs :P :+1:

Reply:
:-) I did a basic test. i.e. Just created those two files, and saw them pop up on the command line for the Ansible process' command line params. If the file is correctly being passed to Ansible, after that it's all up to Ansible anyway.

Reply:
My bit of ruby experience tells me this loop is textbook ruby. ;]